lib/fixtures/user-data/quickstart_with_prop_panel.json

```diff
@@ -10,7 +10,12 @@
       },
       "propertiesPanel": {
         "open": true,
-        "width": 420
+        "width": 420,
+        "groups": {
+          "ElementTemplates__CustomProperties": {
+            "open": true
+          }
+        }
```

Comment:
@jarekdanielak The properties panel layout can be defined in config.json.

Reply:
Cf. https://github.com/camunda/camunda-docs-modeler-screenshots/pull/117/commits/6805e5dc430ca194df53f7620a5c2cb4635b7ba0